packages/components/psammead-brand/src/index.jsx

```diff
@@ -45,6 +46,11 @@ const StyledWrapper = styled.div`
   }
 `;
 
+const ConstraintWrapper = styled.div`
+  max-width: ${GEL_GROUP_5_SCREEN_WIDTH_MIN};
```

Comment:
This adds an extra div to the DOM. Could you move this into`StyledWrapper` above, to prevent the need for extra DOM elements?

Reply:
I dont think so, as the red background goes with it and causes white space on the sides after 1280px